packages/integrations/netlify/package.json

```
@@ -23,6 +23,7 @@
     "./netlify-functions.js": "./dist/netlify-functions.js",
     "./edge-functions": "./dist/integration-edge-functions.js",
     "./netlify-edge-functions.js": "./dist/netlify-edge-functions.js",
+    "./builders": "./builders.d.ts",
```

Maybe this can be `/builders-types` to not mix up with the other `/functions` and `/edge-functions` variants of adapters.
```suggestion
    "./builders-types": "./builders.d.ts",
```